Create hotspot

jkt628 opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
document changes proposed in https://github.com/home-assistant/cli/pull/533 to create an access point




## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/cli/pull/533
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Documentation**
  - Updated command line interface documentation
  - Added new command examples for core updates and network configuration
  - Expanded help command output with additional commands

- **New Features**
  - Enhanced Home Assistant CLI with more comprehensive command list
  - Added ability to update core to specific version
  - Introduced new network and system management commands
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

jkt628 committed:
Create an access point

andrewsayre reviewed (changes requested):
There are unrelated commits in this change. Please reset, rebase, and force push, or close this PR and open a new one. Thank you!

jkt628 commented:
i built HAOS with changes from https://github.com/home-assistant/supervisor/pull/5577 and https://github.com/home-assistant/cli/pull/533 and was able to actually test this on rpi4b.  the command worked with one small tweak so i fixed, squashed my commits, and rebased [next](https://github.com/home-assistant/home-assistant.io/tree/next).

i will hold this `Ready for review` until the other PRs merge and i  can test with official image.

jkt628 committed:
add Practical Note to Create an Access Point

jkt628 commented:
requires https://github.com/home-assistant/operating-system/pull/3865

jkt628 commented:
https://deploy-preview-37140--home-assistant-docs.netlify.app/common-tasks/os/#create-an-access-point

frenck commented:
Closing PR as the parent PR has been closed.
